allennlp/predictors/decomposable_attention.py

```diff
@@ -1,9 +1,11 @@
-
+from typing import List, Dict
+from copy import deepcopy
+import numpy
```

Comment:
Import formatting.  Keep two blank lines after all of the imports and before the class, also.